twitter/api.py

```diff
@@ -179,10 +179,10 @@ def __init__(self,
         # check to see if the library is running on a Google App Engine instance
         # see GAE.rst for more information
         if os.environ:
-			if 'Google App Engine' in os.environ.get('SERVER_SOFTWARE', None):
-				import requests_toolbelt.adapters.appengine  # Adapter ensures requests use app engine's urlfetch
-				requests_toolbelt.adapters.appengine.monkeypatch()
-				cache = None  # App Engine does not like this caching strategy, disable caching
+            if 'Google App Engine' in os.environ.get('SERVER_SOFTWARE', None):
```

Comment:
In this case the value for SERVER_SOFTWARE is actually different depending on the version. Ie. ' 'Google App Engine/3.25' or whatever it might be, so I can't check the whole string.